$NetBSD: patch-al,v 1.3 2023/04/30 14:58:58 spz Exp $

don't complain if the file descriptors limit is higher than what we asked for

--- lib/fdlimit.c.orig	2022-07-10 15:00:35.000000000 +0200
+++ lib/fdlimit.c	2022-09-25 13:21:04.073161149 +0200
@@ -56,6 +56,10 @@
     if (getrlimit(RLIMIT_NOFILE, &rl) < 0) {
         rl.rlim_cur = 0;
         rl.rlim_max = 0;
+    } else {
+        /* will we mind if the limit is higher? I think not. */
+        if (rl.rlim_cur >= limit)
+            return 0;
     }
 #    endif
 
